_episodes/01-introduction.md

```diff
@@ -1,155 +1,127 @@
-> > command line. But, we are working on a user-friendly python interface as
-> > well.
+> > &#10003; **A community effort**
 > >
-> > &#10003; **A way to make climate science more
-> > [FAIR](https://fair-software.eu/about).** ESMValTool collects provenance
-> > information about the data and code that are used to obtain a result. It
-> > comes with a readable recipe format that makes climate analysis consistent,
-> > reproducible, and easy to share.
-> >
-> > &#10005;  **Perfect.** Although we are continuously working to improve the
-> > tool, you may encounter some bugs or missing features. In the following
-> > episodes, you will learn how to troubleshoot, find help, and maybe even
-> > contribute to the solution yourself.
-> >
-> > &#10005;  **Suitable for (Jupyter) notebooks.** ESMValTool was designed as a
-> > command line tool. But, we are working on a user-friendly Python interface
-> > as well.
 > {: .solution}
 {: .challenge}
 
-To learn more about ESMValTool, you can look at the section
-``Where can I get more information on ESMValTool?`` in the lesson
-[Conclusion of the basic tutorial]({{ page.root }}{% link _episodes/07-conclusions.md %}).
+## A tool to analyse climate data
 
-## How does ESMValTool work
+ESMValTool takes care of finding, opening, checking, fixing, concatenating, and
+preprocessing CMIP data and several other supported datasets.
 
-The figure below shows the different components of ESMValTool. Some of the most
-important work is done in the 'core', which performs a number of preprocessor
-steps. Outside of the core we have some configuration files that specify things
-like *where to find the CMIP data*. The most important file however, is the
-*recipe* that specifies which preprocessor functions need to be applied to what
-data. The recipe also points to a diagnostic script that is executed after the
-preprocessor and performs a more specific analysis on the preprocessed data.
+The central componnent of ESMValTool that we will see in this tutorial is the
+**recipe**. Any ESMValTool recipe is basically a set of instructions to reproduce
+a certain result. The basic structure of a recipe is as follows:
 
-![figure showing ESMValTool architecture]({{ page.root
-}}/fig/esmvaltool_architecture.png)
+- **Description** with relevant (citation) information
+- **Datasets** that should be analysed
+- **Preprocessor** steps that must be applied
+- **Diagnostic** scripts performing more specific evaluation steps
 
-> ## Discussion: (dis)advantages of this approach
->
-> Discuss or think about the pros and cons of this architecture for a moment.
-> Then unfold the box below to see our answers.
+An example recipe could look like this:
+
+```yaml
+documentation:
+  description: Example recipe
+  authors:
+    - lastname_firstname
+
+datasets:
+  - {dataset: HadGEM2-ES, project: CMIP5, exp: historical, mip: Amon, ensemble: r1i1p1, start_year: 1960, end_year: 2099}
```

Comment:
This dataset is from the future? ;-)

Perhaps change either experiment or end year.